recipe/meta.yaml

```diff
@@ -43,30 +49,27 @@ test:
     {% for executable in bzip2_executables %}
     - test -f ${PREFIX}/bin/{{ executable }}  # [unix]
     {% endfor %}
-    - test -f ${PREFIX}/include/bzlib.h  # [unix]
-    - test -f ${PREFIX}/lib/libbz2.a  # [unix]
-    - test -f ${PREFIX}/lib/libbz2.so  # [linux]
-    - test -f ${PREFIX}/lib/libbz2.dylib  # [osx]
-    - if not exist %LIBRARY_INC%\\bzlib.h exit 1  # [win]
-    - if not exist %LIBRARY_LIB%\\bzip2.lib exit 1  # [win]
-    - if not exist %LIBRARY_BIN%\\bzip2.dll exit 1  # [win]
-    - if not exist %LIBRARY_LIB%\\libbz2.lib exit 1  # [win]
-    - if not exist %LIBRARY_BIN%\\libbz2.dll exit 1  # [win]
-    - if not exist %LIBRARY_LIB%\\bzip2_static.lib exit 1  # [win]
-    - if not exist %LIBRARY_LIB%\\libbz2_static.lib exit 1  # [win]
+    - test -f ${PREFIX}/include/bzlib.h       # [unix]
+    - test -f ${PREFIX}/lib/libbz2.a          # [unix]
+    - test -f ${PREFIX}/lib/libbz2.so         # [linux]
+    - test -f ${PREFIX}/lib/libbz2.dylib      # [osx]
```

Comment:
```suggestion
    - test -f ${PREFIX}/lib/libbz2.${SHLIB_EXT}        # [unix]
```

Reply:
It will lead to an error because of `/lib/libbz2..so`. I'll fix it